plugins/api-docs/src/components/ApiDefinitionCard/ApiDefinitionCard.tsx

```diff
@@ -28,7 +28,7 @@ type Props = {
 };
 
 export const ApiDefinitionCard = (_: Props) => {
-  const entity = useEntity().entity as ApiEntity;
+  const entity = useEntity<ApiEntity>().entity;
```

Comment:
This is something I didn't know till now! How about
```suggestion
  const {entity} = useEntity<ApiEntity>();
```

Reply:
How can I know about it 🤦 You just implemented it in this PR.... 😆

Reply:
Applied that pattern all over now though (destructuring)